rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
I forgot that I didn't add a match rule for input devices, as I had none to test with... What you'll want to do is match on the `input` subsystem for the device itself, instead of `iio` (or the driver!), and on `ID_INPUT_ACCELEROMETER==1` for that device as well, and match for `usb|acpi|i2c` for the parent device.

This should make sure that, in your case, `/devices/LNXSYSTM:00/LNXSYBUS:00/ACCE0001:00/input/input8` gets tagged.

We might need a slight modification in iio-sensor-proxy to handle that, we can follow-up on its own issues tracker.